src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int AMOUNT = 4;
+    private static final int FIGURE_PARAMETERS = 5;
+    private static final int DEFAULT_FIGURE_PARAMETERS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private Random random = new Random();
+
+    public Figure getRandomFigure(Figure figure) {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/204099572-9c688607-0e30-4ccb-8a58-7d6ad1d6c6f0.png)

Method getRandomFigure() should not get parameters. It should create and return new random figure

Reply:
Does method getRandomFigure() generate not figures with random properties? What it means "shoud not get parameters"? Do I need to generate ramdon values inside this method? Does it not return a new random figure? please explain.

Reply:
1. Please change: "public Figure getRandomFigure(Figure figure)" to "public Figure getRandomFigure()"
2. "....drawFigure()" is not needed in this method. You need to create new random figure and RETURN it